manageiq-providers-ansible_tower.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "ansible_tower_client", "~> 0.19"
+  s.add_runtime_dependency "ansible_tower_client", ">= 0.19.1"
```

Comment:
This change is worth a comment... It didn't seem wise to use `~> 0.19.1` since that would allow only patch updates and it seemed like a lot of effort to do `~> 0.19, >= 0.19`, so I just changed it to allow anything greater than 0.19.1.  We can always be more cautious if we needed to be in the future.